src/image_go_nord_client/main.py

```diff
@@ -0,0 +1,76 @@
+#! /usr/bin/env python3
+
+import logging
+import sys
+from typing import Union
+
+from ImageGoNord import GoNord
+
+from image_go_nord_client import get_argument_parser, get_palette_dict
+
+__ALL__ = ["main"]
+print(sys.warnoptions)
```

Comment:
What is it for? https://docs.python.org/3/library/sys.html#sys.warnoptions

Reply:
I don't remeber, I'm removing it. Maybe for warning purposes.